catalogs/catalog-messaging-kafka/src/main/java/com/datastrato/gravitino/catalog/kafka/KafkaCatalogOperations.java

```diff
@@ -92,39 +113,187 @@ public void initialize(Map<String, String> config, CatalogInfo info) throws Runt
                     e -> e.getKey().substring(CATALOG_BYPASS_PREFIX.length()),
                     Map.Entry::getValue));
     adminClientConfig.putAll(bypassConfigs);
-    adminClientConfig.put(BOOTSTRAP_SERVERS, config.get(BOOTSTRAP_SERVERS));
+    adminClientConfig.put(
+        AdminClientConfig.BOOTSTRAP_SERVERS_CONFIG, config.get(BOOTSTRAP_SERVERS));
     // use gravitino catalog id as the admin client id
-    adminClientConfig.put("client.id", config.get(ID_KEY));
+    adminClientConfig.put(AdminClientConfig.CLIENT_ID_CONFIG, config.get(ID_KEY));
```

Comment:
If we have more than one kafka catalog. Will the id be duplicated?

Reply:
It won't happen. 

I belive each catalog id is unique in Gravitino, right? @yuqi1129

Reply:
I misunderstood. It is recommended to use gravitino+catalogname as the name, which facilitates problem localization.

Reply:
The catalog name may change, but the id won't, so I think having the id here is enough

Reply:
added